generators/kubernetes/templates/registry/_application-configmap.yml

```diff
@@ -30,15 +30,32 @@ data:
     <%_ if (serviceDiscoveryType === 'eureka') { _%>
     eureka:
       client:
-        service-url:
-          # This must contain a list of all Eureka server replicas for registry HA to work correctly
-          defaultZone: http://admin:${jhipster.registry.password}@jhipster-registry-0.jhipster-registry.<%= kubernetesNamespace %>.svc.cluster.local:8761/eureka/,http://admin:${jhipster.registry.password}@jhipster-registry-1.jhipster-registry.<%= kubernetesNamespace %>.svc.cluster.local:8761/eureka/
+        initial-instance-info-replication-interval-seconds: 15
+        instance-info-replication-interval-seconds: 15
+        registry-fetch-interval-seconds: 15
+        fetch-registry: true
+        register-with-eureka: true
+        eureka-service-url-poll-interval-seconds: 15
+      instance:
+        lease-renewal-interval-in-seconds: 10
+        registry-default-open-for-traffic-count: 0
     <%_ } _%>
     jhipster:
       security:
         authentication:
           jwt:
             secret: <%= jwtSecretKey %>
-  #application-prod.yml: |-
-  #  configserver:
-  #    status: Connected in prod
+  # app specific configuration
+  <%_ if (serviceDiscoveryType === 'eureka') {  _%>
+  <%= app.baseName.toLowerCase() %>.yml: |-
+    eureka:
+      client:
+        service-url:
+          # This must contain a list of all Eureka server replicas for registry HA to work correctly
+          defaultZone: http://admin:${jhipster.registry.password}@jhipster-registry.<%= kubernetesNamespace %>.svc.cluster.local:8761/eureka/
```

Comment:
Is `.svc.cluster.local` a standard domain name suffix in Kubernetes?

Reply:
Yes for service objects